cmd/commands/common.go

```diff
@@ -253,6 +254,10 @@ func createAppSet(o *createAppSetOptions) ([]byte, error) {
 		},
 	}
 
+	if (o.appAnnotations != nil) {
```

Comment:
it's okay to just set it on the above object like we do with labels. `nil` would just disappear when running `Marshal`

Reply:
fixed